### DISCUSSION TITLE
I must click on the taskbar or on the desktop to make the menu desseaper (On 2 PCs: Win 10 and Win 11 pro). Any idea to resolved this issue?

### DISCUSSION THREAD

**Discussion by renerelire** [Open-Shell/Open-Shell-Menu#1633] [Q&A] [answered]
Hi, all! When I place the pointer (of my mouse) outside the start button or outside the menu it remains displayed. It does not disappear. I must click on the taskbar or on the desktop to make it go away. 

Example: When I put the pointer over the start button, the menu appears perfectly, but if I don't click on anything inside the menu and put the pointer outside of it, nothing happens. The menu stays open instead going away.

I searched for similar discussions, but I may have not written the right key words.

**Comment by ge0rdi** [comment, marked as answer]
The behavior you are describing is correct and intended one.

If you show start menu (by whatever means - mouse click, Win key press, or hovering on start button if you have it configured so) it will stay until dismissed explicitly (either select some action in menu, or press start button, or click elsewhere, etc).

I guess this is to prevent accidental unwanted menu closing.

**Reply by renerelire** [reply]
Thanks for your help, geOrdi.
